client/FileUpload.js

```diff
@@ -82,23 +102,29 @@ class FileUpload extends Component {
     render() {
         const classes = this.props.classes;
         const doc = this.state.fileDoc;
+        // tags are different in an image doc vs the bio photo in the contrib doc
+        const thumbnail = (this.state.isBioPhoto) ? 'bioThumbnail' : 'thumbnail';
+        const url = (this.state.isBioPhoto) ? 'bioUrl' : 'url';
+        const icon = (this.state.isBioPhoto) ? 'bioIcon' : 'icon';
```

Comment:
Don't need thumbnail and url if they're being set to the same thing - feel free to open an issue to fix this for other Images as well

Reply:
I tried simply removing the thumbnail and that broke the image preview on the chooser fab. I will open an issue to fix this but I'll leave the thumbnail/url code here for now